pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -1598,7 +1598,7 @@ public void testGetTxnState() throws Exception {
         transaction = pulsarClient.newTransaction().withTransactionTimeout(1, TimeUnit.SECONDS)
                 .build().get();
         pulsarServiceList.get(0).getTransactionMetadataStoreService()
-                .endTransaction(transaction.getTxnID(), 0, false);
```

Comment:
Why change this from `commit (0)` to `abort(1)`?
Could you please explain it?

Reply:
The better way is to call `get` here.

Reply:
fixed.